harness/determined/profiler.py

```diff
@@ -15,7 +15,7 @@
 
 SYSTEM_METRIC_TYPE_ENUM = "PROFILER_METRIC_TYPE_SYSTEM"
 TIMING_METRIC_TYPE_ENUM = "PROFILER_METRIC_TYPE_TIMING"
-
+MAX_COLLECTION_SECONDS = 300
```

Comment:
Why are we moving this to a global variable? I think it is quite likely that this is something we will want to configure and so it should remain a property of the `ProfilingAgent`

Reply:
I think making it a property is easy enough once we want it configurable; it was just easier for me to find this way.